content/en/blog/_posts/2019-9-17-SIG-Scalability

```diff
@@ -0,0 +1,26 @@
+SIG Scalability: Learning Kubernetes the Hard Way
+By: Alex Handy
+
+There is a strain of thinking in the software development community, that pushes for the most difficult and rigorous possible method of learning a new language or system. These tend to go by the moniker of “Learn __ the Hard Way.” Examples abound: Learn Code the Hard Way, Learn Python the Hard Way, and of course the one that really started the whole movement, Zed Shaw’s courses in the topic.
+
+While there are folks out there who will offer you a “Learn Kubernetes the Hard Way” type experience--most notably Google’s Kelsey Hightower. That particular example is tied to one cloud in particular, however, and in the spirit of this thing, any “Hard Way” project should probably at least attempt to cover every aspect of the platform being taught.
```

Comment:
This paragraph is a bit confusing and could be removed (the first paragraph establishes context quite well!) 

```suggestion
```